sdf.md

```diff
@@ -1816,10 +1818,73 @@ The initial set of registrations is described in {{sdftype-r}}.
 Security Considerations {#seccons}
 =======================
 
-Some wider issues are discussed in {{-seccons}}.
-
-(Specifics: TBD.)
-
+Some wider security considerations applicable to Things are discussed
+in {{-seccons}}.
+{{Section 5 of -cddl}} gives an overview over security considerations
+that arise when formal description techniques are used to govern
+interoperability; analogs of these security considerations can apply
+to SDF.
+
+The security considerations of underlying building blocks such as
+those detailed in {{Section 10 of -utf8}} apply.
+SDF uses JSON as a representation language; for a number of
+cases {{-json}} indicates that implementation behavior for certain constructs
+allowed by the JSON grammar is unpredictable.
+Implementations need to be robust against invalid or unpredictable
+cases on input, preferably by rejecting input that is invalid or
+that would lead to unpredictable behavior, and need to avoid generating
+these cases on output.
```

Comment:
What should implementations do when they see invalid or unpredictable input?  Should the entire message be dropped (do we have a clear error indication in all mappings?  Do we have mappings for errors?), or just the construct which was invalid, assuming that the format allows for that?
It also might be useful to distinguish code that runs at development time from code that might be generated.

Reply:
I think this is very hard to answer in general.
There are several instances in the text that discuss validity, but we don't have a complete validity model.
What do you do when you get an ABNF spec that is invalid?
I'll merge this PR without having fully addressed your comment; if you think we need to develop a validity model first before going ahead with this document, please add another issue.